iceoryx_hoofs/include/iceoryx_hoofs/cxx/algorithm.hpp

```diff
@@ -122,6 +122,8 @@ doesContainValue(const T value, const T firstValueListEntry, const ValueList...
 /// @param[in] v1 the first sorted input container
```

Comment:
Please add the subscript operator in the line above as well.

Reply:
@FerdinandSpitzschnueffler Not needed anymore as will become private API without doxygen documentation.